package-lock.json

```diff
@@ -19190,7 +19190,6 @@
         "concat-stream": "2.0.0",
         "convert-source-map": "2.0.0",
         "duplexify": "4.1.3",
-        "json-stable-stringify": "1.1.1",
```

Comment:
at first wondered why `packages/browserify/examples/02-scuttler-js/package-lock.json` still has this

(turns out the examples on browserify16, when bumped to browserify17 it'll be gone